client/blocks/reader-combined-card/index.jsx

```diff
@@ -26,7 +26,7 @@ import { getPostsByKeys } from 'state/reader/posts/selectors';
 import ReaderPostOptionsMenu from 'blocks/reader-post-options-menu';
 import PostBlocked from 'blocks/reader-post-card/blocked';
 
-class ReaderCombinedCardComponent extends React.Component {
+class ReaderCombinedCardComponent extends React.PureComponent {
```

Comment:
Does this have to be Pure? Isn't redux taking care of this for us?

Reply:
Taking some measurements it seems that it does indeed take care of this for us, since I'm not seeing any meaningful difference. Thanks for pointing that out!

I'll make the change.